### PR TITLE
Cs account pt01 03

### PR DESCRIPTION
**Pull request by csonparote**
```
Feature: Create account with new user

@PF_CreateAccount @Full
Scenario: Create new account with valid details
Given User is on home page
When User begins account creation
And creates new credentials
And enters confirmation code
And enters valid details
And confirms details
Then Validate that new account is created
#Navigate to home page

@PF_CreateAccountInvalidEmail
Scenario: Create new account with invalid email id
Given User is on home page
When User begins account creation
And User enters invalid email
Then Validate error message is displayed``
`````

### CODE REVIEW

File: platform/src/main/java/platform/pageobjects/AccountServices/CreateAccountStep1.java [hunk 1]
```diff
@@ -68,4 +72,32 @@ public void generateNewCredentials() {
...
+
+	public void enterInvalidEmail() {
+		emailField.click();
+		emailField.sendKeys("abc@abc");
+
+	}
+
+	public void validateErrorMessage(WebElement element, String expectedString) {
+
+		// span[contains(text(),'メールアドレスは、メールアドレス形式で入力してください。')]
```
Comment (Sopan-Patil) [inline platform/src/main/java/platform/pageobjects/AccountServices/CreateAccountStep1.java]: Please remove unwanted comment code

File: platform/src/main/java/platform/pageobjects/AccountServices/CreateAccountStep5.java [hunk 2]
```diff
@@ -66,4 +69,30 @@ public void getCreatedAccountDetails() {
...
+	public void writeCredentialsToExcel() throws IOException {
+
+		ArrayList<String> newLoginData = new ArrayList<>();
+
+		newLoginData.add(0, CreateAccountStep1.email);
+		newLoginData.add(1, CreateAccountStep3.password);
+		newLoginData.add(2, CreateAccountStep1.eid);
+		// = { CreateAccountStep1.email, CreateAccountStep3.password,
+		// CreateAccountStep1.eid };
+		// XLHandler.writeToExcel("NewUser", "NewTestData.xlsx", 0,
```
Comment (Sopan-Patil) [inline platform/src/main/java/platform/pageobjects/AccountServices/CreateAccountStep5.java]: Please remove unwanted comment code

File: platform/src/test/java/platformfeatures/PF_CreateAccount.feature [hunk 3]
```diff
@@ -11,3 +11,9 @@ And confirms details
 Then Validate that new account is created
 #Navigate to home page
```
Comment (Sopan-Patil) [inline platform/src/test/java/platformfeatures/PF_CreateAccount.feature]: remove commneted step

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java [hunk 4]
```diff
@@ -45,11 +49,27 @@ public void user_begins_account_creation() throws Throwable {
...
+		LoginPage loginPage = new LoginPage(driver);
+		loginPage.loginWithNewUser();
+
+		Assert.assertTrue(loginPage.logoutButton.isDisplayed());
+		if (CommonFunctions.isElementVisible(loginPage.logoutButton)) {
+
+			log.info("New User is logged in");
+			loginPage.logoutButton.click();
+		}
+		// Assert.assertTrue(loginPage.logoutButton.isDisplayed());
```
Comment (Sopan-Patil) [inline platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: Remove commented code

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java [hunk 5]
```diff
@@ -90,4 +110,21 @@ public void enters_confirmation_code() throws Throwable {
 
 	}
 
-}
+	@Then("^Validate error message is displayed$")
+	public void validate_error_message_is_displayed() throws Throwable {
+		// throw new PendingException();
+
+		CreateAccountStep1 createAccountStep1 = new CreateAccountStep1(driver);
+		createAccountStep1.validateErrorMessage(createAccountStep1.errorMessageText, "メールアドレスは、メールアドレス形式で入力してください。");
```
Comment (Sopan-Patil) [inline platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: content should not be hard coded.

**Review by Sopan-Patil — CHANGES_REQUESTED** [review]

File: platform/src/main/java/platform/pageobjects/AccountServices/CreateAccountStep1.java (hunk 1 above)
Comment (csonparote) [reply platform/src/main/java/platform/pageobjects/AccountServices/CreateAccountStep1.java]: done

File: platform/src/main/java/platform/pageobjects/AccountServices/CreateAccountStep5.java (hunk 2 above)
Comment (csonparote) [reply platform/src/main/java/platform/pageobjects/AccountServices/CreateAccountStep5.java]: done

File: platform/src/test/java/platformfeatures/PF_CreateAccount.feature (hunk 3 above)
Comment (csonparote) [reply platform/src/test/java/platformfeatures/PF_CreateAccount.feature]: done

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java (hunk 4 above)
Comment (csonparote) [reply platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: done

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java (hunk 5 above)
Comment (csonparote) [reply platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: done

**Comment by csonparote** [comment]
done necessary changes. Please review changed files

**Review by Sopan-Patil — APPROVED** [review]

**Pull request merged**